sprint1.2.css

```
@@ -0,0 +1,481 @@
+*{box-sizing: border-box;
+flex-wrap: wrap;}
```

💊 제안
현재 코드에서는 { 다음 줄에서 속성이 바로 시작되는데, 일반적으로 줄바꿈하여 정렬하면 코드의 가독성이 좋아지고 유지보수하기 쉬워집니다! 이런 스타일을 일관되게 유지하는 것을 추천드려요~

```suggestion
* {
  box-sizing: border-box;
  flex-wrap: wrap;
}
```